contents.json

```diff
@@ -5375,6 +5375,12 @@
          "category": "style-guides",
          "description": "This style guide is based on Apple’s excellent Swift standard library style and also incorporates feedback from usage across multiple Swift projects within Google.",
          "homepage": "https://google.github.io/swift/"
+     },
+     {
+       	"title": "paper-switch",
```

Comment:
@serhii-londar remove extra white spaces please.

Reply:
Done. Also removed all before.

Reply:
@serhii-londar thank you!